packages/connectivity/connectivity/README.md

```diff
@@ -103,6 +103,32 @@ To request location authorization, make sure to add the following keys to your _
 * `NSLocationAlwaysAndWhenInUseUsageDescription` - describe why the app needs access to the user’s location information all the time (foreground and background). This is called _Privacy - Location Always and When In Use Usage Description_ in the visual editor.
 * `NSLocationWhenInUseUsageDescription` - describe why the app needs access to the user’s location information when the app is running in the foreground. This is called _Privacy - Location When In Use Usage Description_ in the visual editor.
 
+### iOS - Disabling code that accesses sensitive data
+
+If you don't wish to set `NSLocationAlwaysAndWhenInUseUsageDescription` and `NSLocationWhenInUseUsageDescription` on iOS, you also have the option to disable the code that uses these permissions.  This may be useful if you don't wish to specify permissions that you have no intention of using and only want to use the plugin for checking the connection status of the device.
+
+To do so, add the following to your `Podfile`:
+```
+post_install do |installer|
+  installer.pods_project.targets.each do |target|
+    flutter_additional_ios_build_settings(target)
+    # Add this section:
+    target.build_configurations.each do |config|
+       config.build_settings['GCC_PREPROCESSOR_DEFINITIONS'] ||= [
+        '$(inherited)',
+        'DISABLE_CONNECTIVITY_LOCATION_CODE=1'
```

Comment:
The location code doesn't matter, you really care about the location permission. This macro is also meant to match the "No Location Permission" in the README.
```
NO_CONNECTIVITY_LOCATION_PERMISSION
```

Reply:
updated to NO_LOCATION_PERMISSION_CONNECTIVITY